src/requestParameters.js

```diff
@@ -1,9 +1,20 @@
-export default function requestParameters (document) {
-  const landing = encodeURI(document.location.href.split('?')[0])
+function findPropertyInParams(params, property) {
+  params.split('&').reduce((acc, param) => {
```

Comment:
Ojo, this is not returning anything and it seems to be modifying the reference variable

Reply:
Also, params can be undefined in case that there's no query params and it will throw an error because you cannot split an undefined

Reply:
Lol